custom_components/adjustable_bed/beds/sleep_number_mcr.py

```
@@ -654,20 +654,35 @@ async def _async_send_frame(
-            frame,
-            cancel_event=cancel_event,
-            response=False,
-        )
+        try:
+            await self._write_gatt_with_retry(
+                SLEEP_NUMBER_MCR_RX_CHAR_UUID,
+                frame,
+                cancel_event=cancel_event,
+                response=True,
+            )
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)